python/cudf/cudf/tests/test_applymap.py

```diff
@@ -44,15 +44,14 @@ def test_applymap_round(nelem, masked):
     np.testing.assert_array_almost_equal(expect, got)
 
 
-def test_applymap_change_out_dtype():
-    # Test for changing the out_dtype using applymap
+def test_apply_change_out_dtype():
+    # Test for changing the out_dtype using apply
 
     data = list(range(10))
 
     sr = Series(data)
 
-    with pytest.warns(FutureWarning):
-        out = sr.applymap(lambda x: float(x), out_dtype=float)
+    out = sr.apply(lambda x: float(x), out_dtype=float)
```

Comment:
I believe this kwarg `out_dtype` has no effect and this test passes by coincidence because `Series.apply` doesn't guard `**kwargs` properly.